docs/components/Intro.js

```diff
@@ -121,200 +122,331 @@ function CliSVG() {
+    </div>
+  );
+};
+
+const displayLinks = (links) => {
+  return links.map((link, i) => {
+    return (
+      <li className="" key={`${link.name} ${i}`}>
+        <div className="landing-card-list-item">
+          {link.book && <BookSVG />}
+          {link.hat && <HatSVG />}
+          {link.cli && <CliSVG />}
+          <Link
+            to={useBaseUrl(link.path)}
+            className="hover:underline font-normal"
+          >
+            {link.name}
+          </Link>
+        </div>
+      </li>
+    );
+  });
+};
+
 function Explained() {
   return (
     <div className="landing-card">
-      <div className="supported-tech">
-        <ul className="landing-card-list-b logos">
-          <li className="list-logo">
-            <a href="https://pkg.go.dev/go.temporal.io/sdk">
-              <img
-                className="w-10 h-8 pl-1 transition hover:scale-110 code-logo"
-                src="/img/golang.svg"
-                alt="Go lang logo"
-              />
-            </a>
-          </li>
-          <li className="list-logo">
-            <a href="https://nodejs.temporal.io">
-              <img
-                className="w-7 h-7 pl-1 transition hover:scale-110 code-logo"
-                src="/img/typescript.svg"
-                alt="TypeScript logo"
-              />
-            </a>
-          </li>
-          <li className="list-logo">
-            <a href="https://github.com/temporalio/sdk-php">
-              <img
-                className="w-10 h-8 pl-1 transition hover:scale-110 code-logo"
-                src="/img/php.svg"
-                alt="PHP logo"
-              />
-            </a>
-          </li>
-          <li className="list-logo">
-            <a href="https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/index.html">
-              <img
-                className="w-7 h-7 pl-1 transition hover:scale-110 code-logo"
-                src="/img/java.svg"
-                alt="Java logo"
-              />
-            </a>
-          </li>
-        </ul>
-      </div>
-      <div className="aside flex">
-        <div className="aside-svg">
-          <InfoSVG />
-        </div>
-        <div className="p-landing">
-          <h3 className="mb-1 aside-head">Temporal Cloud is early access</h3>
-          <p className="mb-0">
-            Access to Temporal Cloud is currently by invitation only.
-          </p>
-          <p>
-            Go to{" "}
-            <a className="aside-link" href={"https://docs.temporal.io/cloud/"}>
-              Cloud docs
-            </a>{" "}
-            or{" "}
-            <a
-              className="aside-link"
-              href="https://pages.temporal.io/cloud-early-access"
-            >
-              join the waitlist
-            </a>
-          </p>
-        </div>
-      </div>
+      <p className="main-subhead">
+        Temporal Platform explanation and usage information.
+      </p>
       <div className="landing-card-section">
```

Comment:
With the Cloud announcement gone, the padding between this div and the subhead above seems excessive to my eye.